src/components/MovieListPage.tsx

```diff
@@ -1,46 +1,23 @@
 import {useState} from "react";
-import {Movie, MovieDialogState, MovieListState} from "../models/movies";
+import {Movie, MovieDialogState} from "../models/movies";
 import Header from "./header/Header";
-import MovieDetails from "./movie-details/MovieDetails";
 import Content from "./content/Content";
-import {Portal} from "react-portal";
 import MovieDialog from "./dialogs/movie-dialog/MovieDialog";
-import {SortByOptions} from "../constants/sort-control-const";
-import useFilteredMovieList from "../hooks/UseFilteredMovieList";
-import {GenreTitle} from "../constants/genres-const";
 import React from "react";
+import useFilteredMovieList from "../hooks/UseFilteredMovieList";
 
 function MovieListPage(): React.JSX.Element {
-    const [state, updateState] = useState<MovieListState>({
-        search: '',
-        sortBy: SortByOptions.releaseDate.value,
-        activeGenre: '',
-        selectedMovie: null,
-    });
     const [movieDialogData, setMovieDialogData] = useState<MovieDialogState>({
         isOpen: false,
         title: '',
         movie: null,
     });
```

Comment:
If you not move this logic out to a custom hook to be reusable, there is no need to use 'isOpen' and 'title'. Just save here only a movie. isOpen === !!movie; title can be a static string